packages/material-ui/src/index.js

```diff
@@ -1,5 +1,21 @@
 import * as colors from './colors';
 
+if (
+  process.env.NODE_ENV !== 'production' &&
```

Comment:
Don't we want the opposite: Only warn for production builds.

Reply:
I have used https://github.com/ant-design/ant-design/blob/859be320fb18555ec54a43f13662917055b4946a/components/index.tsx#L3-L15 as a baseline. Are they doing it wrong?

Reply:
That file assumes that the plugin is used:
> // this file is not used if use https://github.com/ant-design/babel-plugin-import

Reply:
Yes, I agree